README.md

```diff
@@ -14,15 +14,35 @@ With `kubicorn` a user can create new clusters, modify and scale them, and take
 **NOTE:** This is a work-in-progress, we do not consider it production ready.
 Use at your own risk and if you're as excited about it as we are, maybe you want to join us on the #kubicorn channel in the Gophers Slack community.
 
+## Installing
+
+### Quickstart
+
+Assuming you have a local Go environment The Simplest method to install is to just run:
+
 ```
 $ go get github.com/kris-nova/kubicorn
 ```
 
+### Not so Quickstart
```

Comment:
Can we *only* have the quickstart on the main readme, and then add a link to an in depth building.md in `/doc`?

I want the main readme to be short/sweet and I have a feeling we are going to have to start documenting the Makefile anyway down the road. What are your thoughts here? I am kind of on the fence.